var/spack/repos/builtin/packages/py-openmc/package.py

```diff
@@ -33,7 +34,17 @@ class PyOpenmc(PythonPackage):
     variant("mpi", default=False, description="Enable MPI support")
 
     # keep py-openmc and openmc at the same version
-    for ver in ["develop", "master", "0.13.1", "0.13.0", "0.12.2", "0.12.1", "0.12.0", "0.11.0"]:
+    for ver in [
+        "develop",
+        "master",
+        "0.13.2",
+        "0.13.1",
+        "0.13.0",
+        "0.12.2",
+        "0.12.1",
+        "0.12.0",
+        "0.11.0",
+    ]:
         depends_on(
             "openmc+mpi@{0}".format(ver), when="@{0}+mpi".format(ver), type=("build", "run")
         )
```

Comment:
Dependencies need to be updated. Not sure when these things changed, but the latest version requires:

* `python@3.7:`
* `numpy@1.9:1.21`

Reply:
Good point -- I've updated the dependency for Python. For numpy, what's currently listed (`@1.9:`) should still apply.

Reply:
According to https://github.com/openmc-dev/openmc/blob/v0.13.2/pyproject.toml, numpy 1.22 is not yet supported.

Reply:
Ah yes, you're right. I've just updated the numpy dependency.